fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountReadPlatformServiceImpl.java

```diff
@@ -791,7 +793,13 @@ public SavingsAccountTransactionsMapper() {
             sqlBuilder.append("left join m_account_transfer_transaction totran on totran.to_savings_transaction_id = tr.id ");
             sqlBuilder.append("left join m_payment_detail pd on tr.payment_detail_id = pd.id ");
             sqlBuilder.append("left join m_payment_type pt on pd.payment_type_id = pt.id ");
-
+			sqlBuilder.append("left join m_appuser au on au.id=tr.appuser_id ");
+			sqlBuilder.append("left join m_note mn on mn.savings_account_id=sa.id ");
+			sqlBuilder.append("left join (");
+            sqlBuilder.append("select mn.savings_account_id id, mn.note Note");
+            sqlBuilder.append(" from m_note mn ");
+            sqlBuilder.append(") comments ON comments.id=tr.savings_account_id  ");
+			
```

Comment:
@nazeer1100126 , we didn't have the skype call but however i tried to work things out. Then challenge i got is that the query is not joining the note with the transaction id correctly. Please have a look at it here before i make changes to send final PR. Thanks
What happens is that it repeats one note for all transactions not particular to a transaction